add codecov docs (#6482)
getsentry/sentry-docs: src/docs/product/codecov/set-up.mdx

---
title: Set Up
sidebar_order: 1
---

> ## Set Up

Author: ```suggestion

```

> If you're a user of both Codecov and Sentry, you can enable the Codecov integration with 1 click in your Organization Settings page in the Sentry app. You'll be able to see whether you have partial or full test coverage for the line of code that's causing an error, directly in your stack trace in Sentry.

Author: ```suggestion
If you're a user of both Codecov and Sentry, you can enable the Codecov integration with 1 click in your **Organization Settings** page in the Sentry app. You'll be able to see whether you have partial or full test coverage for the line of code that's causing an error, directly in your stack trace in Sentry.
```

> ### Prerequisites

Author: ```suggestion
## Prerequisites
```

There are a few prerequisites that have to be fulfilled in order for Codecov to work in Sentry. You have to have:

- Owner or manager-level permissions in Sentry.io.

> - The [Github Integration](/integrations/source-code-mgmt/github/) installed in Sentry with [Code Mappings](/integrations/source-code-mgmt/github/#stack-trace-linking) enabled.

Author: ```suggestion
- The [Github Integration](/product/integrations/source-code-mgmt/github/) installed in Sentry with [Code Mappings](/product/integrations/source-code-mgmt/github/#stack-trace-linking) enabled.
```

>     If you have the Github integration set up correctly, your stack traces will show an “Open this line in Github” message.
> </Note>

Author: ```suggestion
    If you have the Github integration set up correctly, your stack traces will show an “Open this line in Github” prompt that will take you to your app's code.
</Note>
```

> - A [Codecov](https://about.codecov.io/) account with [Source Code Coevreage](https://about.codecov.io/product/feature/source-code-coverage/) enabled.

Author: ```suggestion
- A [Codecov](https://about.codecov.io/) account with [Source Code Coverage](https://about.codecov.io/product/feature/source-code-coverage/) enabled.
```

- A [Codecov](https://about.codecov.io/) account with [Source Code Coevreage](https://about.codecov.io/product/feature/source-code-coverage/) enabled.

> ## Enable Codecov insights

Author: ```suggestion
## Enable Codecov Insights
```

> Once the prerequisites have been met, you can enable Codecov by going to the **Settings** page in Sentry and toggling “Enable Code Coverage Insights” to on.
> ![Codecov in Org Settings](codecov-setup-in-ui.png)

Author: ```suggestion
Once the prerequisites have been met, you can enable Codecov by going to the **Settings** page in Sentry and toggling “Enable Code Coverage Insights” to on.

![Codecov in Org Settings](codecov-setup-in-ui.png)
```
